tests/expressions/test_queryset_values.py

```diff
@@ -28,13 +28,11 @@ def test_values_expression(self):
         )
 
     def test_values_expression_group_by(self):
-        # values() applies annotate() first, so values selected are grouped by
-        # id, not firstname.
         Employee.objects.create(firstname='Joe', lastname='Jones', salary=2)
         joes = Employee.objects.filter(firstname='Joe')
         self.assertSequenceEqual(
             joes.values('firstname', sum_salary=Sum('salary')).order_by('sum_salary'),
-            [{'firstname': 'Joe', 'sum_salary': 2}, {'firstname': 'Joe', 'sum_salary': 10}],
```

Comment:
This test checked unexpected behavior (at least for me). Previously
```python
joes.values('firstname', sum_salary=Sum('salary')).order_by('sum_salary')
```
and 
```python
joes.values('firstname').annotate(sum_salary=Sum('salary'))
```
returned different results because in the first queryset we added `sum_salary` to the `GROUP BY` clause. Now in both cases we group by `firstname`.